src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -728,36 +704,29 @@ private StandardizationHistory setCurrentStandardizationDryRunStatus(String stat
 
 	@Override
 	public void executeDryRun() throws CmpdRegMolFormatException, IOException, StandardizerException {
-
-		StandardizationHistory stndznHistory = this.setCurrentStandardizationDryRunStatus("running");
+		logger.info("standardization dry run initialized");
```

Comment:
This function is what is actually executed by the API route. Its job is to reset the standardization table (truncate it), create a new standardization history and set it to running. and then kick off a `checkForDryRunStandardization` which starts to populate/dupe check the dry run table.